CHANGELOG.md

```diff
@@ -4,12 +4,14 @@
 
 ### Template
 
+- Turn on automatic clean up of intermediate files in `work/` on successful pipeline completion in full-test config ([#2163](https://github.com/nf-core/tools/pull/2163))
 - Turn on automatic clean up of intermediate files in `work/` on successful pipeline completion in full-test config ([#2163](https://github.com/nf-core/tools/pull/2163)) [Contributed by @jfy133]
 - Add documentation to `usage.md` on how to use `params.yml` files, based on nf-core/ampliseq text ([#2173](https://github.com/nf-core/tools/pull/2173/)) [Contributed by @jfy133, @d4straub]
 - Make jobs automatically resubmit for a much wider range of exit codes (now `104` and `130..145`) ([#2170](https://github.com/nf-core/tools/pull/2170))
 - Remove problematic sniffer code in samplesheet_check.py that could give false positive 'missing header' errors ([https://github.com/nf-core/tools/pull/2194]) [Contributed by @Midnighter, @jfy133]
 - Consistent syntax for branch checks in PRs ([#2202](https://github.com/nf-core/tools/issues/2202))
 - Fixed minor Jinja2 templating bug that caused the PR template to miss a newline
+- Added `[skip ci]` to nf-core linting bot to stop executing entire pipeline CI tests for file formatting changes ([2171](https://github.com/nf-core/tools/pull/2171)
```

Comment:
uh... no you didn't? 😅

Reply:
(interpreting `[skip ci]` in a fairly strict way as meaning in a commit message)

Reply:
It was before, and then we've dropped it - I've not yet updated the CHANGELOG as it seems we would be updating the system... (it was too simplistic the first time around)